how-to/4--self-host-workspace/README.md

```diff
@@ -0,0 +1,54 @@
+# Self Host OpenFin Workspace
+
+This example showcases how to self host OpenFin Workspace's frontend assets.
+
+## Getting Started
+
+1. Install dependencies.
+
+```bash
+$ npm install
+```
+
+2. Build the project.
+
+```bash
+$ npm run build
+```
+
+3. Download the sample self hosted build of OpenFin Workspace.
+
+```bash
+$ npm run download
+```
+
+4. Start the sample server.
+
+```bash
+$ npm run start
+```
+
+5. Launch Home and Browser using the locally served assets.
+   (Make sure Home and Browser are closed before executing command.)
```

Comment:
should we add the script we have in the other example that shuts down all OF processes?